fastjet/jet.go

```diff
@@ -10,11 +10,11 @@ import (
 	"go-hep.org/x/hep/fmom"
 )
 
-// // Used to protect against parton-level events where pt can be zero
-// // for some partons, giving rapidity=infinity. KtJet fails in those cases.
-// const (
-// 	MaxRap = 1e5
-// )
+// Used to protect against parton-level events where pt can be zero
+// for some partons, giving rapidity=infinity. KtJet fails in those cases.
+const (
+	MaxRap = 1e5
```

Comment:
actually, I think the doc would render better if the doc comment was put just on top of `MaxRap` instead of the `const` block.
(it shows that I wrote that part a couple of years ago :))